CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ The format is based on [Keep a Changelog] and this project adheres to
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed recursion in `rustc_link_search!(…)`.
+- Removed redundant `$x:literal` macro patterns.
+
+### Added
+
+- 40 snapshot tests (using [insta](https://crates.io/crates/insta))
+- Optional `to:` parameter for writing into a `std::fmt::Write` or `io::fmt::Write`.
```

Comment:
Please document the `to:` parameter with an doctest example in the docs of all the macros that take `to:`.

Reply:
Done.